tools/run_tests/xds_k8s_test_driver/tests/outlier_detection_test.py

```diff
@@ -29,11 +29,21 @@
 _XdsTestServer = xds_k8s_testcase.XdsTestServer
 _XdsTestClient = xds_k8s_testcase.XdsTestClient
 
+# Testing consts
+_QPS = 100
+_REPLICA_COUNT = 5
 
 class OutlierDetectionTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
 
+    @staticmethod
+    def is_supported(config: skips.TestConfig) -> bool:
```

Comment:
Missing an import and an alias for _Lang - this will break things unless fixed.

Reply:
Fixed.